src/redux/store.js

```diff
@@ -0,0 +1,14 @@
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import booksSlice, { addBook } from './books/booksSlice';
+import categoriesSlice from './categories/categoriesSlice';
+
+const rootReducers = combineReducers({
+  booksReducers: booksSlice,
+  categoriesReducer: categoriesSlice,
+});
+
+const store = configureStore({
+  reducer: rootReducers,
+});
+
+store.dispatch(addBook({ id: 1, item: 'new boooks' }));
```

Comment:
- [ ] Please consider removing this action dispatch as I assume you were testing the functionality. 🚀 🤓